libraries.gradle

```diff
@@ -2,30 +2,31 @@
 // use.  In that respect it serves a role similar to <dependencyManagement> in Maven
 ext {
     vavrVersion = '0.10.0'
-    rxJavaVersion = '2.2.7'
-    reactorVersion = '3.2.6.RELEASE'
+    rxJavaVersion = '2.2.8'
+    reactorVersion = '3.2.8.RELEASE'
     junitVersion = '4.12'
-    slf4jVersion = '1.7.24'
-    assertjVersion = '3.6.2'
-    logbackVersion = '0.9.26'
-    mockitoVersion = '2.15.0'
-    powermockVersion = '2.0.0-beta.5'
-    jcacheVersion = '1.0.0'
+    slf4jVersion = '1.7.26'
+    assertjVersion = '3.12.2'
+    logbackVersion = '1.2.3'
+    mockitoVersion = '2.25.1'
+    powermockVersion = '2.0.0'
+    jcacheVersion = '1.1.0'
     awaitilityVersion = '1.7.0'
     metricsVersion = '3.2.6'
-    vertxVersion = '3.5.1'
-    aspectjVersion = '1.8.13'
-    spring4Version = '4.3.22.RELEASE'
+    vertxVersion = '3.7.0'
+    aspectjVersion = '1.9.2'
+    springVersion = '5.1.6.RELEASE'
     springBootVersion = '1.5.19.RELEASE'
```

Comment:
@RobWin can you update this to `1.5.20` also, please? 😃   
It has been released since I opened my PR.

Reply:
done